src/Microsoft.Azure.WebJobs.Host/Config/FluentConverterRules.cs

```diff
@@ -16,7 +17,7 @@ namespace Microsoft.Azure.WebJobs.Host.Config
     public abstract class FluentConverterRules<TAttribute, TThis> where TAttribute : Attribute
     {
         // Access the converter manager that we're adding rules to. 
-        internal abstract IConverterManager Converters { get; }
+        internal abstract ConverterManager Converters { get; }
```

Comment:
Why not use the interface?

Reply:
The public interface is very constrained (just a Getter). Whereas the class is private, and it has additional setters. I'm trying to shirnk the public surface area as much as possible to reduce liability. 

---
In reply to: [148663519](https://github.com/Azure/azure-webjobs-sdk/pull/1413#discussion_r148663519) [](ancestors = 148663519)